native/packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBar.ios.js

```diff
@@ -52,20 +52,21 @@ const statusBarPadding = isIphoneX ? 44 : 20;
 const styles = StyleSheet.create({
   barOuter: {
     flexDirection: 'column',
-    paddingHorizontal: 8,
+    paddingHorizontal: 8, // eslint-disable-line backpack/use-tokens
     width: '100%',
     backgroundColor: colorGray50,
     shadowColor: colorGray100,
     shadowOpacity: 1,
     shadowRadius: 0,
-    shadowOffset: { width: 0, height: 1 },
+    shadowOffset: { width: 0, height: 1 }, // eslint-disable-line backpack/use-tokens
   },
   barOuterWithSubtitle: {
-    paddingBottom: 16,
+    paddingBottom: 16, // eslint-disable-line backpack/use-tokens
   },
   barInner: {
     paddingTop: statusBarPadding, // Status bar
-    height: 64, // 44 for the bar + 20 for the status bar
+    // 44 for the bar + 20 for the status bar
+    height: 44, // eslint-disable-line backpack/use-tokens
```

Comment:
Looks like a typo - it's changed from `64` to `44`?

Reply:
fixing it 😞

Reply:
done